config/default.yml

```diff
@@ -18,6 +18,10 @@ linters:
   ControlStatementSpacing:
     enabled: true
 
+  EmbeddedEngines:
+    enabled: true
```

Comment:
Can we simply disable this if we aren't forbidding any engines?

Reply:
Yes, it makes sense.
I will fix this as suggested!